web/react/stores/channel_store.jsx

```diff
@@ -209,10 +209,6 @@ var ChannelStore = assign({}, EventEmitter.prototype, {
     catch (err) {
 	}
 
-	if (channels == null) {
-		channels = [];
-	}
```

Comment:
~~Yes, probably should have moved my comment about it from the big block of text about it specifically to here but it keeps channels uninitialized if it fails to get it from browser storage so that  some stuff in a separate commit that was merged before the stuff I just deleted works (a loading animation shows in the More Channel menu when the list of "more channels" still hasn't been loaded.  So with it being left uninitialized~~

Yes it was intentional but the fact that you pointed it out shows that it's kind of a "scary" solution.  I therefore changed how I'm checking whether the list of "more channels" to a better check that uses the .loading property to see if the list of more channels exists yet.